engines/sci/engine/vm.h

```diff
@@ -136,6 +136,21 @@ enum {
 	VAR_PARAM = 3
 };
 
+enum GlobalVar {
+	kEgo            = 0,
+	kGame           = 1,
+	kCurrentRoom    = 2,
+	kSpeed          = 3,  // SCI16
+	kDefaultPlane   = 3,  // SCI32
+	kQuit           = 4,
+	kPlanes         = 10, // SCI32
+	kCurrentRoomNo  = 11,
+	kPreviousRoomNo = 12,
+	kNewRoomNo      = 13,
+	kFastCast       = 84, // SCI16
+	kMessageType    = 90
+};
+
```

Comment:
Shouldn't those be better called kGlobalVarSpeed and such?
I don't really like calling those "kQuit" etc. because you don't know immediately that those are GlobalVar enums.

I'm also not sure if it's wise to mix sci16 and sci32 together, because all sorts of things are not the same.
So maybe: "kGlobalVar16Speed" and so on. For the mixed ones, one could drop the 16/32.